source/MRMesh/MRMeshDelone.cpp

```diff
@@ -174,8 +175,14 @@ int makeDeloneEdgeFlips( Mesh & mesh, const DeloneSettings& settings, int numIte
         if ( progressCallback && !progressCallback( float( iter ) / numIters ) )
             return flipsDone;
 
+        UndirectedEdgeBitSet flipCandidates;
+        flipCandidates.resize( mesh.topology.undirectedEdgeSize() );
```

Comment:
Can we move `flipCandidates` construction before the `for ( int iter = 0; iter < numIters; ++iter )`?

Reply:
Fixed.